internal/gengapic/example.go

```diff
@@ -131,6 +131,62 @@ func (g *generator) exampleMethod(pkgName, servName string, m *descriptor.Method
 	return nil
 }
 
+func (g *generator) exampleMethodBody(pkgName, servName string, m *descriptor.MethodDescriptorProto) error {
```

Comment:
Makes sense! Let me know if the way I refactored it makes sense; I kept the 
```
if m.GetClientStreaming() != m.GetServerStreaming() {
		// TODO(pongad): implement this correctly.
		return nil
	}
```
in both `exampleMethod` and `exampleMethodBody` as that would break both methods, but otherwise I think the code is deduplicated.

Reply:
SGTM